terraform/01-jenkins-server/outputs.tf

```diff
@@ -0,0 +1,22 @@
+output "instance_id" {
+  description = "ID of the EC2 instance"
+  value       = aws_instance.jenkins.id
+}
+
+output "private_ip" {
+  description = "Private ip of the EC2 instance"
+  value       = aws_instance.jenkins.private_ip
+}
+
+output "public_ip" {
+  description = "Public ip of the EC2 instance"
+  value       = aws_instance.jenkins.public_ip
+}
+
+output "ec2_user" {
```

Comment:
ec2_user it's value, not name of EC2 instance variable ;)

Reply:
Changed